content/docs/command-reference/add.md

```diff
@@ -6,8 +6,8 @@ Track data files or directories with DVC, by creating a corresponding
 ## Synopsis
 
 ```usage
-usage: dvc add [-h] [-q | -v] [-R] [--no-commit] [-f <filename>]
-               targets [targets ...]
+usage: dvc add [-h] [-q | -v] [-R] [--no-commit] [--external]
+               [-f <filename>] targets [targets ...]
 
 positional arguments:
   targets               Input files/directories to add.
```

Comment:
```suggestion
  targets               Files or directories to add
```
Would imply a change in the core repo too though.

Reply:
👍

Reply:
Moved to https://github.com/iterative/dvc.org/pull/1411#pullrequestreview-426717410